README.md

```
@@ -1,2 +1,45 @@
+  > feature/add-login
+  > fix/issue-123
+
+Do you want to continue? (Y/n) y
+
+Deleting 2 branches...
+Deleted 2 branches.
+```
+
+## Contributing
+
+Contributions are welcome! Please feel free to submit a pull request or open an issue if you have any feedback or suggestions.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)